src/components/AdminPanelNavigation/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { CloseOutlined } from '@mui/icons-material';
+import { IconButton } from '@mui/material';
+import { Box } from '@mui/system';
+import { useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import { useFetchAllSalonsQuery } from 'store/api/admin';
+import { CustomDrawer, CustomToolbar } from './adminPanelNavigation.styled';
+import SalonsNavigation from './adminPanelNavigation.list';
+
+const AdminPanelNavigation = ({ isMenuOpen, handleClose }: { isMenuOpen: boolean; handleClose: () => void }) => {
+  const location = useLocation();
+  const { data = [] } = useFetchAllSalonsQuery();
+
+  useEffect(() => handleClose(), [location]);
+
+  return (
+    <CustomDrawer open={isMenuOpen} variant="temporary">
+      <CustomToolbar>
+        <IconButton onClick={handleClose}>
+          <CloseOutlined />
+        </IconButton>
+      </CustomToolbar>
+      <Box p={2}>
```

Comment:
If you want to use that mystical acronym, please leave at least a comment on what it means 🙂